plyer/tests/test_audio.py

```diff
@@ -0,0 +1,58 @@
+'''
+TestAudio
+=========
+
+Tested platforms:
+
+* macOS
+'''
+
+import unittest
+import time
+
+from os import mkdir, remove
+from os.path import join, expanduser, exists
+from plyer.tests.common import platform_import, PlatformTest
+
+
+class TestAudio(unittest.TestCase):
+    '''
+    TestCase for plyer.audio.
+    '''
+
+    @PlatformTest('macosx')
+    def test_audio_macosx(self):
+        '''
+        Test macOS audio start, stop and play
+        '''
+        path = join(expanduser('~'), 'Music')
+        if not exists(path):
+            mkdir(path)
+
+        audio = platform_import(
+            platform='macosx',
+            module_name='audio',
+        )
+
+        self.assertIn('OSXAudio', dir(audio))
+        audio = audio.instance()
+        self.assertIn('OSXAudio', str(audio))
+
+        self.assertIsNone(audio.start())
```

Comment:
Make sure `audio.file_path` does not exist before you start the recording.

    self.assertFalse(exists(audio.file_path))